docker/Dockerfile

```diff
@@ -61,7 +61,28 @@ RUN OS_ARCH=${TARGETARCH/x86_64/amd64} && OS_ARCH=${OS_ARCH/aarch64/arm64} && \
     curl -LO https://dl.k8s.io/release/$(curl -L -s https://dl.k8s.io/release/stable.txt)/bin/linux/${OS_ARCH}/kubectl && \
     chmod +x ./kubectl
 
-FROM nvcr.io/nvidia/k8s/cuda-sample:vectoradd-cuda12.5.0 AS sample-builder
+FROM nvcr.io/nvidia/cuda:12.9.1-base-ubi9 AS sample-builder
+
+RUN dnf install -y --allowerasing \
```

Comment:
I can never understand why the folks at `dnf` never followed the naming conventions and named the CLI flag `--allow-erasing` instead